config/menus/weapon_vars.cfg

```diff
@@ -0,0 +1,606 @@
+                                        
+                                        ui_list [
+                                            ui_textfield $cur_variable_name; ui_strut 1
+                                            variable_reset_button
+                                        ]
+                                        
+                                    ] $VAR_TYPE_STRING [
+                                    
+                                        ui_text         "^faVariable Type: ^fwString"
+                                        ui_text (format "^faVariable Name: ^fw%1" $cur_variable_name)
+                                        ui_strut 0.5
+                                        
+                                        ui_text (format "^fgDefault: ^fw%1" $cur_variable_value_def) "" -1 -1 1000
+                                        ui_strut 1
+                                        
+                                        ui_textfield $cur_variable_name 45
+                                        ui_strut 1
+                                        
+                                        variable_reset_button
+                                        
+                                    ] () [ 
+                                        ui_text "^faVariable Type: ^fw???"
+                                    ]
+                                ]
+                            ]
+                            
+                            ui_strut 0 1
+                            
+                            ui_font "emphasis" [ ui_text "Info:" ]
+                            ui_text $cur_variable_description "" -1 -1 1200
+                            
+                            scurfmax = (- (<< 1 $cur_variable_fields) 1)
+                            if (& (= $cur_variable_type $VAR_TYPE_INTEGER) (> $cur_variable_fields 1)) [
+                                if (!= $scurfmax $cur_variable_value_max) [
+                                    ui_text (format "^fyUsage description of %1 implies a max value of %2, but it should be %3." $cur_variable_name $scurfmax $cur_variable_value_max)
+                                ]
+                                ui_list [
+                                    ui_strut 5
+                                    ui_list [
+                                        loop i $cur_variable_fields [
+                                            ui_font "little" [ ui_bitfield (getvarfields $cur_variable_name $i) $cur_variable_name (<< 1 $i) ]
+                                        ]
+                                    ]
+                                ]
+                            ]
+                        ] ]
+                    ]
+                ]
+            ]
+
+        ]
+    ]
+]
+newgui weap_vars [
+    ui_header "Weapons"
+    ui_list [ weapmenu ]
+] [
+    if (= $guipasses 0) [
+        fill_variable_lists
+    ]
+]
+
+bind F7 [ showgui weap_vars ]
```

Comment:
This bind should be in setup.cfg with the rest of the binds.
F7 is already bound there to the team selector, though that is also bound to PERIOD, so there is some choice how to proceed.
There should also be a way to access this menu from the menu system.

Reply:
any suggestions where to put it?

Reply:
Probably around or in the current variables menu.

Reply:
yeah, that probably makes a lot of sense, will bind the entire menu to F7 so one can easily access the other variables while they're at it too